frontend/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:22-slim@sha256:bac8ff0b5302b06924a5e288fb4ceecef9c8bb0bb92515985d2efdc3a2447052
+
+WORKDIR /app
+
+RUN corepack enable
+RUN corepack prepare pnpm@10.7.0 --activate
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)